modules/Web/sites/slots/config-authsettingsv2/README.md

```diff
@@ -1,4 +1,4 @@
-# Site Config `[Microsoft.Web/sites/slots/config-authsettingsv2]`
+# Site Config `[Microsoft.Web/sites/slots/configAuthsettingsv2]`
```

Comment:
For this to work we should rename the module replacing `'-'` with `'--'` already, since the readme logic already includes code for the next filter renaming and we need to differentiate the two. If you're fine reviewing it I can add it to this PR.

Reply:
I think that was something we've been shortly debating with the PG to, i.e. that `config-hello` would become `config---hello` if we'd keep it as is. Can't say I'm the biggest fan to continue on adding these `-` :D 
So before we do anything ... do you think we should consider alternatives like `config_hello` instead?

Reply:
I'm not sure `_` would be accepted by the brm tool. A double `--` is anyway fine and would be enough to distinguish. The problem is that the rename of those folders will increase the review effort, as the resulting readme diff would be the entire file instead of just the title. An alternative would be to keep it as is and do a third PR managing only those `config-hello` folders. What would you prefer?

Reply:
Hola @eriqua, my only concern was/is that `--` looks very similar to `-`, especially when comparing the following examples:

EventGrid
- `modules\DigitalTwins\digitalTwinsInstances\endpoints-eventGrid\main.bicep`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints--eventGrid\main.bicep`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints---eventGrid\main.bicep`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints_eventGrid\main.bicep`

EventHub
- `modules\DigitalTwins\digitalTwinsInstances\endpoints-eventHub\main.bicep`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints--eventHub\main.bicep`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints---eventHub\main.bicep`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints_eventHub\main.bicep`

ServiceBus
- `modules\DigitalTwins\digitalTwinsInstances\endpoints-serviceBus`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints--serviceBus`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints---serviceBus`
- `modules\DigitalTwins\digitalTwinsInstances\endpoints_serviceBus`

However, I don't have a strong opinion about it. I guess the only downside I'm currently seeing with `---` is that it is 3 characters we loose, lengthwise.

Any any case - I don't think the changes will be too bad to review as I commented all modules where it is relevant already. Without checking I think it is about 10 or so and I already reviewed the files. So only those will be marked as 'changed since last review' anyways. :)

Reply:
done